### PR TITLE
Cms Kit - code mirror for page script & style

### LINKED ISSUE #11934: Use code-mirror for CMS-Kit Page script/style
According to https://github.com/abpframework/abp/pull/11885

### PR DESCRIPTION
**Pull request by enisn**
Closes https://github.com/abpframework/abp/issues/11934

---

According to https://github.com/abpframework/abp/pull/11885

---

## Testing Locally
Add those following blocks temporarily since npm packages haven't been released yet for testing on **Volo.CmsKit.Web.Unified** project.

- **package.json**
  ```json
      "@abp/codemirror": "~5.2.0-rc.1"
  ```
- **abp.resourcemapping.js**
  ```js
      mappings: {
          "@node_modules/codemirror/lib/*.*": "@libs/codemirror/",
          "@node_modules/codemirror/mode/**/*.*": "@libs/codemirror/mode/",
          "@node_modules/codemirror/theme/**/*.*": "@libs/codemirror/theme/",
          "@node_modules/codemirror/addon/**/*.*": "@libs/codemirror/addon/"
      }
  ```

---

image
image

### CODE REVIEW

File: modules/cms-kit/host/Volo.CmsKit.Web.Unified/abp.resourcemapping.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ module.exports = {
         "@libs"
     ],
     mappings: {
-
+        "@node_modules/codemirror/lib/*.*": "@libs/codemirror/",
+        "@node_modules/codemirror/mode/**/*.*": "@libs/codemirror/mode/",
+        "@node_modules/codemirror/theme/**/*.*": "@libs/codemirror/theme/",
+        "@node_modules/codemirror/addon/**/*.*": "@libs/codemirror/addon/"
```
Comment (rqx110) [inline modules/cms-kit/host/Volo.CmsKit.Web.Unified/abp.resourcemapping.js]: no need these mappings, it has config in @abp/codemirror package

File: modules/cms-kit/host/Volo.CmsKit.Web.Unified/package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@abp/aspnetcore.mvc.ui.theme.basic": "^5.2.0-rc.1",
-    "@abp/cms-kit": "5.2.0-rc.1"
+    "@abp/cms-kit": "5.2.0-rc.1",
+    "@abp/codemirror": "~5.2.0-rc.1"
```
Comment (rqx110) [inline modules/cms-kit/host/Volo.CmsKit.Web.Unified/package.json]: no need add @abp/codemirror, it has reference by @abp/cms-kit (by @abp/cms-kit.admin internal)

Comment (enisn) [reply modules/cms-kit/host/Volo.CmsKit.Web.Unified/package.json]: it's just for testing, because they haven't released yet

Comment (enisn) [reply modules/cms-kit/host/Volo.CmsKit.Web.Unified/package.json]: I shouldn't push those I think

**Review by yekalkan — APPROVED** [review]

**Pull request merged**